releasenotes/notes/deprecate-register-deregister-cmd.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: release-notes/v2
+kind: deprecate-istioctl-cmd
```

Comment:
same as https://github.com/istio/istio/pull/29593#discussion_r542111625

Reply:
yes, thanks @shamsher31! I also aware of that, will update the word to removed